Gems/Multiplayer/Code/Source/AutoGen/AutoComponent_Header.jinja

```diff
@@ -176,7 +176,7 @@ RewindableFixedVector<{{ Property.attrib['Type'] }}, {{ Property.attrib['Count']
 AZStd::fixed_vector<{{ Property.attrib['Type'] }}, {{ Property.attrib['Count'] }}> m_{{ LowerFirst(Property.attrib['Name']) }};
 {% endif %}
 {%     elif Property.attrib['IsRewindable']|booleanTrue %}
-Multiplayer::RewindableObject<{{ Property.attrib['Type'] }}, Multiplayer::RewindHistorySize> m_{{ LowerFirst(Property.attrib['Name']) }} = {{ Property.attrib['Init'] }};
+Multiplayer::RewindableObject<{{ Property.attrib['Type'] }}, Multiplayer::RewindHistorySize> m_{{ LowerFirst(Property.attrib['Name']) }} = { {{ Property.attrib['Init'] }} };
```

Comment:
There should not be an "=" at the end here.
End it with just:
.... LowerFirst(Property.attrib['Name']) }} { {{ Property.attrib['Init'] }} };

Reply:
done